resources/fixtures/Briteq/Briteq-BT-575S.qxf

```diff
@@ -180,15 +180,15 @@
   <Channel Number="2">Shutter shaking</Channel>
   <Channel Number="3">Gobo</Channel>
   <Channel Number="4">Color 2</Channel>
-  <Channel Number="5">Gobo rotation</Channel>
+  <Channel Number="5" ActsOn="3">Gobo rotation</Channel>
   <Channel Number="6">Prism/Prism rotation/Dimmer</Channel>
   <Channel Number="7">Focus</Channel>
  </Mode>
  <Physical>
-  <Bulb Type="MSD 575W" Lumens="0" ColourTemperature="6500"/>
-  <Dimensions Weight="27.7" Width="427" Height="478" Depth="39"/>
-  <Lens Name="PC" DegreesMin="14" DegreesMax="14"/>
-  <Focus Type="Fixed" PanMax="540" TiltMax="270"/>
-  <Technical PowerConsumption="600" DmxConnector="3-pin"/>
+  <Bulb Type="" Lumens="0" ColourTemperature="0"/>
```

Comment:
All the physical information got lost. Please fix

Reply:
Apologies. It will be fixed.